.eslintrc.yml

```diff
@@ -39,5 +39,4 @@ rules:
   ]
   sort-keys: [2, asc, {caseSensitive: true, natural: true}]
   sort-vars: 2
-  spaced-comment: [2, always, {exceptions: ['#']}]
```

Comment:
Shouldn't we rather add the `/` to solve the issue and still have a rule for this? -> ``spaced-comment: [2, always, {exceptions: ['#', '/']}]`